app/components/ask-travis.js

```diff
@@ -40,6 +40,7 @@ export default Component.extend({
       console.log('TOGGLE');
```

Comment:
this looks like some debugging / development log, if it is no longer needed please remove it